visualization/app/codeCharta/codeCharta.service.ts

```diff
@@ -31,6 +34,7 @@ export class CodeChartaService {
 				this.addFile(nameDataPair.fileName, nameDataPair.content)
 			} catch (e) {
 				if (!_.isEmpty(e.error)) {
+					this.storeService.dispatch(setIsLoadingFile(false))
```

Comment:
`this.fileStates` has to be reset here to zero. Otherwise multiple files could still fail.